### DISCUSSION TITLE
Travel guide with Visitsights partnership

### DISCUSSION THREAD

**Discussion by Nic787** [organicmaps/organicmaps#2559] [Ideas]
Hi,

I remember on Maps.me before it stalled, it has a travel guide feature where people were able to create and share some itineraries.

I think it could be nice to return into OrganicMaps. I found Visitsights the other day. They offer some self-guided city tour, but they don't have any apps and doesn't work offline. I was thinking it could be great to implement some of their guide into the app and maybe allow users to create their own city guide again?

Thank you

**Comment by biodranik** [comment]
An interesting idea, thanks. Their tours can be supported by adding GPX files import (which is planned).
